packages/favorites/tests/FavoriteHeart.test.js

```diff
@@ -95,8 +96,8 @@ avSettingsApi.getApplication = getApplicationMock;
 describe('FavoriteHeart', () => {
   beforeEach(() => {
     global.document.createRange = () => ({
-      setStart: () => {},
-      setEnd: () => {},
+      setStart: () => ({}),
+      setEnd: () => ({}),
```

Comment:
Is this change intentional? One is a function that does nothing and returns undefined. The other is a function that returns an empty object.